jupyter-nbconvert 和 python-pytest-jupyter 打包

目前 python-jupyter-client 和 jupyter-server 的一些测试还过不去, 可以先考虑把 jupyter-nbconvert 和 python-pytest-jupyter 打出来. (jupyter-nbcovert 卡了很多包)

jupyter 这边依赖图太乱了, 先让 `jupyter-nbconvert` 和 `python-pytest-jupyter` 进仓库也能简化一下后续的打包.

## 处理 jupyter-nbconvert

先打一版 nocheck

```
extra-riscv64-build -- -d "$CACHE_DIR:/var/cache/pacman/pkg/" -- --nocheck
```

## 处理 python-jupyter-client

它依赖 python-ipykernel

注意 python-ipykernel 也依赖了 `python-jupyter_client`. 需要先 `pacman -S python-ipykernel --assume-installed python-jupyter_client`

然后 `makepkg -s --nocheck` 即可出一版 nocheck 的包

## 处理 jupyter-server

先打一版 nocheck

```
extra-riscv64-build -- -d "$CACHE_DIR:/var/cache/pacman/pkg/" -I ../../../python-jupyter-client-8.2.0-1-any.pkg.tar.zst -I ../../../jupyter-nbconvert-7.2.10-1-any.pkg.tar.zst -- --nocheck
```

## 处理 python-pytest-jupyter

```
extra-riscv64-build -- -d "$CACHE_DIR:/var/cache/pacman/pkg/" -I ../../../jupyter-server-2.5.0-1-any.pkg.tar.zst -I ../../../jupyter-nbconvert-7.2.10-1-any.pkg.tar.zst -I ../../../python-jupyter-client-8.2.0-1-any.pkg.tar.zst
```

## 处理 jupyter-nbconvert

注意这里需要把以前打出来的 nocheck 包塞进去.  否则测试会失败. (该包测试时依赖了已经安装在系统中的该包的文件)

```
extra-riscv64-build -- -d "$CACHE_DIR:/var/cache/pacman/pkg/" -I ../../../python-jupyter-client-8.2.0-1-any.pkg.tar.zst -I ../../../jupyter-nbconvert-7.2.10-1-any.pkg.tar.zst
```

---

这后面的可以暂时不打包

---


## 处理 python-jupyter-client

```
extra-riscv64-build -- -d "$CACHE_DIR:/var/cache/pacman/pkg/" -I ../../../python-jupyter-client-8.2.0-1-any.pkg.tar.zst -I ../../../python-pytest-jupyter-0.7.0-1-any.pkg.tar.zst -I ../../../jupyter-server-2.5.0-1-any.pkg.tar.zst -  
I ../../../jupyter-nbconvert-7.2.10-1-any.pkg.tar.zst
```

目前会有一个测试挂掉, 还没找到原因(https://github.com/jupyter/jupyter_client/issues/946):

```
tests/test_client.py::TestAsyncKernelClient::test_input_request FAILED   [ 16%]
```

## 处理 jupyter-server

```
extra-riscv64-build -- -d "$CACHE_DIR:/var/cache/pacman/pkg/" -I ../../../python-pytest-jupyter-0.7.0-1-any.pkg.tar.zst -l kxxt2 -I ../../../jupyter-server-2.5.0-1-any.pkg.tar.zst  -I ../../../python-jupyter-client-8.2.0-1-any.pkg  
.tar.zst -I ../../../jupyter-nbconvert-7.2.10-1-any.pkg.tar.zst
```

Arch 上游把测试跳过去了, 进 chroot 手动跑测试:

```
cd /build/jupyter-server/src/jupyter_server-2.5.0
pytest -v
```


<details>

<summary> 目前有 49 个测试挂掉  </summary>

```
FAILED tests/test_terminal.py::test_no_terminals - tornado.httpclient.HTTPClientError: HTTP 404: Not Found  
FAILED tests/test_terminal.py::test_terminal_create - tornado.httpclient.HTTPClientError: HTTP 404: Not Found  
FAILED tests/test_terminal.py::test_terminal_create_with_kwargs - tornado.httpclient.HTTPClientError: HTTP 404: Not Found  
FAILED tests/test_terminal.py::test_terminal_create_with_cwd - tornado.httpclient.HTTPClientError: HTTP 404: Not Found  
FAILED tests/test_terminal.py::test_culling_config - KeyError: 'terminal_manager'  
FAILED tests/test_terminal.py::test_culling - tornado.httpclient.HTTPClientError: HTTP 404: Not Found  
FAILED tests/test_terminal.py::test_shell_command_override[shell_command="['/path/to/shell', '-l']"-expected_shell0-5.4] - KeyError: 'terminal_manager'  
FAILED tests/test_terminal.py::test_shell_command_override[shell_command="/string/path/to/shell -l"-expected_shell1-5.1] - KeyError: 'terminal_manager'  
FAILED tests/test_terminal.py::test_importing_shims - ModuleNotFoundError: No module named 'jupyter_server_terminals'  
FAILED tests/auth/test_authorizer.py::test_authorized_requests[True-DELETE-/api/contents/{nbpath}-None] - assert 500 in {200, 201, 204, None}  
FAILED tests/auth/test_authorizer.py::test_authorized_requests[True-POST-/api/terminals-] - KeyError: 'terminal_manager'  
FAILED tests/auth/test_authorizer.py::test_authorized_requests[True-GET-/api/terminals-None] - KeyError: 'terminal_manager'  
FAILED tests/auth/test_authorizer.py::test_authorized_requests[True-GET-/terminals/websocket/{term_name}-None] - KeyError: 'terminal_manager'  
FAILED tests/auth/test_authorizer.py::test_authorized_requests[True-DELETE-/api/terminals/{term_name}-None] - KeyError: 'terminal_manager'  
FAILED tests/auth/test_authorizer.py::test_authorized_requests[False-POST-/api/terminals-] - KeyError: 'terminal_manager'  
FAILED tests/auth/test_authorizer.py::test_authorized_requests[False-GET-/api/terminals-None] - KeyError: 'terminal_manager'  
FAILED tests/auth/test_authorizer.py::test_authorized_requests[False-GET-/terminals/websocket/{term_name}-None] - KeyError: 'terminal_manager'  
FAILED tests/auth/test_authorizer.py::test_authorized_requests[False-DELETE-/api/terminals/{term_name}-None] - KeyError: 'terminal_manager'  
FAILED tests/extension/test_app.py::test_stop_extension - AssertionError: assert {'tests.exten...ckextensions'} == {'jupyter_ser...ckextensions'}  
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager--inroot] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-Directory with spaces in-inspace] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-unicod\xe9-innonascii] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo-a] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo-b] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo-name with spaces] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo-unicod\xe9] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo/bar-baz] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-ordering-A] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-ordering-b] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-ordering-C] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-\xe5 b-\xe7 d] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager--inroot] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-Directory with spaces in-inspace] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-unicod\xe9-innonascii] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo-a] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo-b] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo-name with spaces] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo-unicod\xe9] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo/bar-baz] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-ordering-A] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-ordering-b] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-ordering-C] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-\xe5 b-\xe7 d] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete_dirs[FileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete_dirs[AsyncFileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete_non_empty_dir[FileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_delete_non_empty_dir[AsyncFileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_regression_is_hidden[FileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
FAILED tests/services/contents/test_api.py::test_regression_is_hidden[AsyncFileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error  
==================================================================================== 49 failed, 826 passed, 15 skipped, 20 warnings in 433.70s (0:07:13) ====================================================================================
```

</details>

> 这部分看起来是个应该也存在于 x86_64 的问题？

Upstreamed: https://bugs.archlinux.org/task/78214

>  jupyter-server 目前有 49 个测试挂掉

都在 qemu-user chroot 里跑的话跟在 x86_64 上挂掉的测试是一样的.  可以认为问题不大, 继续出包.

如果在 riscv64 板子上, chroot 之外测试, 目前挂掉 33 个测试.

<details>
<summary> Failing tests on riscv64 </summary>

```
FAILED tests/auth/test_authorizer.py::test_authorized_requests[True-DELETE-/api/contents/{nbpath}-None] - assert 500 in {200, 201, 204, None}
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager--inroot] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-Directory with spaces in-inspace] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-unicod\xe9-innonascii] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo-a] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo-b] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo-name with spaces] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo-unicod\xe9] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo/bar-baz] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-ordering-A] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-ordering-b] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-ordering-C] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-\xe5 b-\xe7 d] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager--inroot] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-Directory with spaces in-inspace] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-unicod\xe9-innonascii] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo-a] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo-b] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo-name with spaces] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo-unicod\xe9] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo/bar-baz] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-ordering-A] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-ordering-b] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-ordering-C] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-\xe5 b-\xe7 d] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete_dirs[FileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete_dirs[AsyncFileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete_non_empty_dir[FileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete_non_empty_dir[AsyncFileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_regression_is_hidden[FileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_regression_is_hidden[AsyncFileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/kernels/test_cull.py::test_cull_idle[jp_server_config0] - tornado.simple_httpclient.HTTPTimeoutError: Timeout during request
FAILED tests/services/kernels/test_cull.py::test_cull_idle[jp_server_config1] - tornado.simple_httpclient.HTTPTimeoutError: Timeout during request
```

</details>

在 x86_64 上, chroot 之外测试的话会挂 34 个测试.


<details>

<summary> Failing tests on x86_64 </summary>

```
FAILED tests/test_terminal.py::test_terminal_create_with_cwd - AssertionError: assert 'terminal_path' in 'pwd\r\n\r\n\x1b[H\x1b[2J\x1b[3JThis is the Z Shell configuration function for new users,\r\nzsh-newuser-install.\r\nYou are seeing this message because you have no zsh startup files\r\n(the...
FAILED tests/test_terminal.py::test_culling - assert False
FAILED tests/auth/test_authorizer.py::test_authorized_requests[True-DELETE-/api/contents/{nbpath}-None] - assert 500 in {200, 201, 204, None}
FAILED tests/nbconvert/test_handlers.py::test_from_file_zip - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager--inroot] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-Directory with spaces in-inspace] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-unicod\xe9-innonascii] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo-a] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo-b] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo-name with spaces] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo-unicod\xe9] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-foo/bar-baz] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-ordering-A] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-ordering-b] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-ordering-C] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[FileContentsManager-\xe5 b-\xe7 d] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager--inroot] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-Directory with spaces in-inspace] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-unicod\xe9-innonascii] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo-a] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo-b] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo-name with spaces] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo-unicod\xe9] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-foo/bar-baz] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-ordering-A] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-ordering-b] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-ordering-C] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete[AsyncFileContentsManager-\xe5 b-\xe7 d] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete_dirs[FileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete_dirs[AsyncFileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete_non_empty_dir[FileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_delete_non_empty_dir[AsyncFileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_regression_is_hidden[FileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
FAILED tests/services/contents/test_api.py::test_regression_is_hidden[AsyncFileContentsManager] - tornado.httpclient.HTTPClientError: HTTP 500: Internal Server Error
```

</details>

--- a/python-jupyter-client/riscv64.patch
+++ b/python-jupyter-client/riscv64.patch
@@ -1,0 +1,11 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -25,7 +25,7 @@ build() {
+ 
+ check() {
+   cd $_pyname-$pkgver
+-  pytest -v
++  pytest -v --deselect tests/test_client.py::TestAsyncKernelClient::test_input_request
+ }
+ 
+ package() {

